lib/rules/no-color-literals.js

```diff
@@ -57,6 +57,9 @@ const create = Components.detect((context) => {
 
 module.exports = {
   meta: {
+    docs: {
+      description: 'Detect `StyleSheet` rules and inline styles containing color literals instead of variables',
```

Comment:
Moved rule descriptions inside the standard `meta.docs.description` field in rules. This allows tooling like eslint-doc-generator to utilize these descriptions.